pertpy/tools/_metadata.py

```diff
@@ -0,0 +1,71 @@
+import anndata
+import pandas as pd
+
+
+class MetaData:
+    """Python implementation of MetaData."""
+
+    def __init__(self):
+        self.cell_line_meta = pd.read_csv("https://ndownloader.figshare.com/files/35020903")
+
+    def annotate_cell_lines(
+        self,
+        adata: anndata,
+        cell_line_column: str = "DepMap_ID",
+        cell_line_type: str = "DepMap_ID",
+        cell_line_information: list = None,
+    ):
+        """Fetch cell line annotation.
+
+        For each cell, we fetch cell line annotation from Dependency Map (DepMap).
+
+        Args:
+            adata: The data object to annotate.
+            cell_line_column: The column of `.obs` with cell line information.
+            cell_line_type: The type of cell line information, e.g. DepMap_ID, cell_line_name or stripped_cell_line_name.
+            cell_line_information: The metadata to fetch. All metada will be fetched by default.
+
+        Returns:
+            Returns an `adata` with cell line annotation.
+        """
+        adata_copy = adata.copy()
```

Comment:
Why do we need a copy? Always try to avoid copies. Consider adding a `copy: bool = False` parameter like some of the other functions in pertpy. Only if this is true we create a copy and return it.